src/main/scala/scorex/core/consensus/History.scala

```diff
@@ -5,7 +5,7 @@ import scorex.core.consensus.History.ProgressInfo
 import scorex.crypto.encode.Base58
 
 import scala.util.Try
-
+// scalastyle:off
```

Comment:
Why do we need to switch it off here? What was it complaining about in the comment? Maybe we should switch only the triggered rule, instead of switching off all rules?

Reply:
It's because of line length limit. I will rework for sure to be more specific here.